pkg/oc/cli/idle/idle.go

```diff
@@ -545,6 +546,24 @@ type scaleInfo struct {
 // scalable resources to zero, and annotating the associated endpoints objects with the scalable resources to unidle
 // when they receive traffic.
 func (o *IdleOptions) RunIdle() error {
+	operatorClient, err := operatorclient.NewForConfig(o.ClientConfig)
```

Comment:
You should instantiate client in `Complete` method, like the other ones are.

Reply:
Not sure I follow. The appsclient is instantiated within RunIdle, not within Complete.

Reply:
> Not sure I follow. 

Look how `ScaleClient` is instantiated and used within this command. That's the blessed way of doing stuff in CLI these days. 

> The appsclient is instantiated within RunIdle, not within Complete.

True, that was there before we setup the new guidelines for all CLI commands. You may move this in this PR or please open an additional one fixing it.

Reply:
I see, will just include the changes on this PR, thanks